action-a/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM debian:9.5-slim
```

Comment:
Notice here is where you'd have used `FROM ros:melodic` to get a base image that provides you ROS